protocol/credentials.go

```diff
@@ -74,6 +116,7 @@ type CredentialsOfferMessageBody struct {
 type CredentialOffer struct {
 	ID          string `json:"id"`
 	Description string `json:"description"`
+	Status      string `json:"status,omitempty"`
```

Comment:
don't see `Status` here https://iden3-communication.io/credentials/1.0/offer/
Is it a new field ?